wellregistry/registry/tests/test_admin.py

```diff
@@ -12,24 +12,49 @@
 
 
 class TestRegistryFormAdmin(TestCase):
-    fixtures = ['test_agencies.json']
+    fixtures = ['test_agencies.json', 'test_countries.json', 'test_counties.json', 'test_states.json', 'test_altitude_datum.json', 'test_horizontal_datum.json', 'test_nat_aquifer.json', 'test_units.json']
```

Comment:
nit: This line is too long. If you just put a line feed in near the middle, that would work. PyCharm should do the formatting for you.